src/internal/mediaUtil.ts

```diff
@@ -85,6 +91,16 @@ export function validateSelectMediaInputs(mediaInputs: media.MediaInputs): boole
   return true;
 }
 
+/**
+ * Returns true if the mediaInput params are called for mediatype Image and contains Image outputs formats false otherwise
```

Comment:
nit: formats `and` false otherwise OR formats`,` false otherwise

Reply:
@ShivangiBeniwal Review comments addressed